ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/CracResultManager.java

```diff
@@ -131,7 +131,7 @@ public void fillCnecLoopFlowExtensionsWithInitialResults(LoopFlowResult loopFlow
                 double loopFlowThreshold = Math.abs(cnecLoopFlowExtension.getInputThreshold(Unit.MEGAWATT, network));
                 double initialLoopFlow = Math.abs(loopFlowResult.getLoopFlow(cnec));
 
-                cnecLoopFlowExtension.setLoopFlowConstraintInMW(Math.max(initialLoopFlow, loopFlowThreshold - cnec.getFrm()));
+                cnecLoopFlowExtension.setLoopFlowConstraintInMW(Math.max(initialLoopFlow + raoData.getLoopFlowAcceptableAugmentation(), loopFlowThreshold - cnec.getFrm()));
```

Comment:
we could instead the LoopFlowAcceptableAugmentation in argument of this method [if we remove it from raoData]

Reply:
It would be nice to have a new unit test for this :angel:

Reply:
:+1: